File: Security/Security Function Inverses/sol.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner in = new Scanner(System.in);
        int size = in.nextInt();
        int[] arr = new int[size];
        for(int i =0; i < size; i++){
            arr[i] = in.nextInt();
        }
        
        int[] inverse = new int[size];
        for(int i = 0 ; i < size; i++){
            int temp = arr[i];
            inverse[temp-1] = i+1;
        }
        
        for(int i = 0 ; i < size ; i++){
            System.out.println(inverse[i]);
        }
    }
}

File: 30 Days of Code/Day 2/sol.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main(int argc,char*argv[]) {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */    
    double price;
    double tip;
    double tax;
    
    scanf("%lf",&price);
    scanf("%lf",&tip);
    scanf("%lf",&tax);
    tip = tip/100;
    tax = tax/100;
    double sum = price + price*tip + price*tax;
    sum = round(sum);
    printf("The total meal cost is %.0lf dollars.",sum);
    
    return 0;
}
